apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/ArrayAsIterableAssertionsSpec.kt

```diff
@@ -28,25 +31,25 @@ class ArrayAsIterableAssertionsSpec : ch.tutteli.atrium.spec.integration.ArrayAs
     Companion::booleanArrayWithCreator
 ){
     companion object {
-        fun arrayInt(plant: Assert<Array<Int>>) = plant.asIterable()
-        fun byteArray(plant: Assert<ByteArray>) = plant.asIterable()
-        fun charArray(plant: Assert<CharArray>) = plant.asIterable()
-        fun shortArray(plant: Assert<ShortArray>) = plant.asIterable()
-        fun intArray(plant: Assert<IntArray>) = plant.asIterable()
-        fun longArray(plant: Assert<LongArray>) = plant.asIterable()
-        fun floatArray(plant: Assert<FloatArray>) = plant.asIterable()
-        fun doubleArray(plant: Assert<DoubleArray>) = plant.asIterable()
-        fun booleanArray(plant: Assert<BooleanArray>) = plant.asIterable()
+        fun arrayInt(plant: Assert<Array<Int>>) = plant.asExpect<Array<out Int>, Assert<Array<out Int>>>().asList(ch.tutteli.atrium.api.infix.en_GB.o).asAssert()
```

Comment:
oh man, that looks ugly :laughing: especially because of `ch.tutteli.atrium.api.infix.en_GB.o`
however, if I checkout your branch and do the replacement, then I get
```
plant.asExpect<Array<out Int>, Assert<Array<out Int>>>().asList(o).asAssert()
```
which is much better. Can you please check if the replacement works for you; i.e. revert the changes in this file and do the replacements again.

Reply:
Yeah I know it's a bit ugly.
Because I can't automatically replace it with `plant.asExpect<Array<out Int>, Assert<Array<out Int>>>().asList(o).asAssert()` I will check this out.

Reply:
Don't spend too much time, it is fine like this as well. I was merely curious why it did not work for you since it worked for me.